stf.load/src/stf.load/net/adoptopenjdk/loadTest/reporting/OutputFilter.java

```diff
@@ -53,8 +53,12 @@ public void write(byte buf[], int off, int len) {
 		ExecutionTracker executionTracker = ExecutionTracker.instance();
 
 		if (executionTracker.isRunningTest()) {
-			// Allow the test adaptor to examine the output, and decide if test is passing/failing
-			executionTracker.checkTestOutput(buf, off, len);
+			try {
+				// Allow the test adaptor to examine the output, and decide if test is passing/failing
+				executionTracker.checkTestOutput(buf, off, len);
+			} catch (MauveTestFailureException e) {
+				e.printStackTrace();
```

Comment:
@Mesbah-Alam it appears this printStackTrace() is causing a StackOverflowError.
```
LT  	at java.lang.Throwable.printStackTrace(Throwable.java:250) ~[?:1.8.0_262-internal]
LT  	at net.adoptopenjdk.loadTest.reporting.OutputFilter.write(OutputFilter.java:60) ~[bin/:?]
LT  	at sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:221) ~[?:1.8.0_262-internal]
LT  	at sun.nio.cs.StreamEncoder.implFlushBuffer(StreamEncoder.java:291) ~[?:1.8.0_262-internal]
LT  	at sun.nio.cs.StreamEncoder.flushBuffer(StreamEncoder.java:104) ~[?:1.8.0_262-internal]
LT  	at java.io.OutputStreamWriter.flushBuffer(OutputStreamWriter.java:185) ~[?:1.8.0_262-internal]
LT  	at java.io.PrintStream.write(PrintStream.java:527) ~[?:1.8.0_262-internal]
LT  	at java.io.PrintStream.print(PrintStream.java:669) ~[?:1.8.0_262-internal]
LT  	at java.io.PrintStream.println(PrintStream.java:806) ~[?:1.8.0_262-internal]
LT  	at net.adoptopenjdk.loadTest.reporting.ExecutionTracker.recordTestFailure(ExecutionTracker.java:161) ~[bin/:?]
LT  	at net.adoptopenjdk.loadTest.LoadTestRunner$2.run(LoadTestRunner.java:232) ~[bin/:?]
LT  	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_262-internal]
LT  	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_262-internal]
LT  	at java.lang.Thread.run(Thread.java:823) [?:1.8.0_262-internal]
```
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_ppc64le_linux_Nightly_mauveLoadTest/60/tapResults/
We didn't get the core dump either, not sure why.

Reply:
About the StackOverflowError, do you see anything in the [Exception class](https://github.com/AdoptOpenJDK/stf/blob/master/stf.load/src/stf.load/net/adoptopenjdk/loadTest/reporting/MauveTestFailureException.java) that may be causing it? If we replace call to `e.printStackTrace()` with System.out.print(e.getMessage()) - would that make any difference? This is a hard problem to test, as the error seems to only get reproduced in live builds - not sure why.

Reply:
I expect calling System.out.print() would cause the same problem. Let's just remove the printStackTrace().

Reply:
> Let's just remove the printStackTrace().

- Ok, will do.